components/avatar/avatar.css

```diff
@@ -0,0 +1,61 @@
+
+.img-avatar{
+    margin:2rem 0 2rem 0;
+  }
+  span{
+    color:#ff758f;
+  }
+  
+  /* avatar */
+  .avatar-round-xlg, .avatar-round-lg, .avatar-round-md, .avatar-round-sm, .avatar-round-xsm{
```

Comment:
You can add a common class 'avatar-round'  and set it to 50% and then use it as below

```
 <img class="avatar-round avatar-xlg" src="../../assets/images/avatar fox.jpg"/>
```